gossipd/handshake.c

```diff
@@ -49,19 +49,6 @@ struct act_one {
 /* BOLT #8: The handshake message is _exactly_ `50 bytes` */
 #define ACT_ONE_SIZE 50 /* ARM's stupid ABI adds padding. */
 
-static inline void check_act_one(const struct act_one *act1)
-{
-	/* BOLT #8:
-	 *
-	 * : `1 byte` for the handshake version, `33 bytes` for the compressed
-	 * ephemeral public key of the initiator, and `16 bytes` for the
-	 * `poly1305` tag.
-	 */
-	BUILD_ASSERT(sizeof(act1->v) == 1);
-	BUILD_ASSERT(sizeof(act1->pubkey) == 33);
-	BUILD_ASSERT(sizeof(act1->tag) == 16);
-}
-
 /* BOLT #8:
```

Comment:
See comments on #426 

We can't remove this.  We can, however, call it, if that makes things better.

Reply:
Yep, calling them is best I think, just to avoid future confusion.